static/js/invite.js

```diff
@@ -70,7 +70,7 @@ exports.initialize = function () {
         },
         success: function () {
             $('#submit-invitation').button('reset');
-            invite_status.text(i18n.t('User invited successfully.',
+            invite_status.text(i18n.t('User(s) invited successfully.',
                                       {count: (invitee_emails.val().match(/@/g) || []).length}))
```

Comment:
Hmm, is this `count` thing trying to do something clever with pluralization? ... Or is it a legacy of some previous version that showed the actual number?

Reply:
Good catch. Legacy of something previous; I've removed it. 
Long term this message should just be replaced with a checkmark animation and closing the invite modal.

Reply:
Makes sense.

Did you have a new version of the branch you meant to push? Doesn't look like it's made it to GitHub.

Reply:
normally I don't reply to comments until I've pushed a new version (in this case the tests surprisingly failed and then I forgot to come back), but the convention I have with tabbott is I explicitly write "Updated" or similar when ready for re-review.

I'm trying to cement this convention with a zulipbot rule, so hopefully this will improve in the future.

Reply:
Cool, happy to follow that convention, then.